packages/inline-definition/README.md

```diff
@@ -52,10 +52,57 @@ npm install @leafygreen-ui/inline-definition
 
 ## Properties
 
-| Prop         | Type                           | Description                                                  | Default |
-| ------------ | ------------------------------ | ------------------------------------------------------------ | ------- |
-| `definition` | **Required** `React.ReactNode` | Content that describes the term. Will appear inside Tooltip. |         |
-| `children`   | `string`                       | Text that will appear underlined                             |         |
-| `className`  | `string`                       | className will be applied to the trigger element             |         |
+| Prop                    | Type              | Description                                                  | Default |
+| ----------------------- | ----------------- | ------------------------------------------------------------ | ------- |
+| `definition` (Required) | `React.ReactNode` | Content that describes the term. Will appear inside Tooltip. |         |
+| `children`              | `string`          | Text that will appear underlined                             |         |
+| `className`             | `string`          | className will be applied to the trigger element             |         |
+| ...                     | `TooltipProps`    | Any other properties will be spread on the Tooltip element   |         |
 
-_Any other properties will be spread on the `Tooltip` element_
+##### TooltipProps
+
+```typescript
+const Justify = {
```

Comment:
what's the intended use case for this section? it's not going to be fun to have to keep it up to date

Reply:
use-case is that we say in the PropTable type is `TooltipProps` but there's no context provided to consumer as to what that means

Reply:
can we do something like `[Tooltip props](link to tooltip prop table)`

Reply:
yeah that works too

Reply:
Done